src/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.csproj

```diff
@@ -1,24 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$(RepoRoot)eng/pkg/Pack.props"/>
-
```

Comment:
nit: keep an empty line after the project opening tag and at the end of the project file before the project closing tag. That's the style that our templates use and which I personally find easier to read.

Reply:
I'll apply this convention across the repo.